src/manual/design/apis/pulse.md

```diff
@@ -10,13 +10,41 @@ communicate between microservices, notifying interested parties when specific
 events have occurred. The system is based on AMQP in a publish-subscribe
 configuration.
 
+## Background
+
+Mozilla, where Taskcluster began, has an organization-wide "message bus" called
+"pulse". The idea is that systems within Mozilla should publish events to this
+bus, and other systems can then consume those events. This has enabled
+development of lots of interesting pieces of automation. It was a natural
+choice to publish Taskcluster-related events to this message bus, and to also
+consume such events from the bus.
+
+That said, Taskcluster does not consume events from other Mozilla services, and
+can use any other AMQP service just as easily. Consider the use of the term
+"pulse" to be a historical artifact.
```

Comment:
How about:
> Consider use of the term _pulse_ to mean using RabbitMQ in according with the [Pulse specification](https://wiki.mozilla.org/Auto-tools/Projects/Pulse#Specification).

Maybe we should even embed the pulse specification in our docs.
Basically, it says to only use `topic` exchanges, publish JSON messages, use durable exchanges, and a few other things like that. There are many ways to use RabbitMQ, _pulse_ is a specific way to use RabbitMQ as a loosely coupled pub-sub system.


The goal of the pulse specification is to avoid the many pit-falls of things that can go wrong with RabbitMQ. If a client follows the pulse spec it's pretty easy to get things right.